plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -112,7 +115,42 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 			waitUntil: 'networkidle',
 		} );
 		await page.locator( '#title' ).fill( nonVirtualProductName );
+
+		await page
+			.frameLocator( '#content_ifr' )
```

Comment:
I was wondering what was going on with this iframe.  Wild that the description textarea is actually contained in one!

Reply:
Yeah :) I couldn't find handling iframes in other `e2e-pw` tests, so if there's a better way of handling it, please, let me know 🙇

Reply:
@Stojdza , @lanej0 I was struggling to get how I could get inside the iframe's text area to add a product description while working on the https://github.com/woocommerce/woocommerce-quality/issues/607.

Is it okay if I reuse these lines from Ivan's solution?

Reply:
Ofc @alvarothomas 👍 Please let me know in case you need any help with it

Reply:
This is the right way to do it! I was more just surprised to see the need for it 😆